site/templates/collection/carousel/item/image.jet

```diff
@@ -11,6 +11,16 @@
         {{focusClass = "focus-center"}}
       {{end}}
     {{end}}
-    <s72-image src="{{.Images.Carousel}}" alt="{{.Title}}" class="carousel-item-image {{focusClass}}"></s72-image>
+
+
+    {{if showTrailerBg}}
+    <video autoplay muted loop playsInline class="carousel-item-video" data-src="{{trailerURL}}" poster="{{.Images.Carousel}}">
```

Comment:
should probably put this into its own jet. its a bit wierd that its in the image.jet